everett/manager.py

```diff
@@ -912,15 +912,24 @@ def __call__(self, key, namespace=None, default=NO_VALUE,
                         raise
                     except Exception as orig_exc:
                         exc_info = sys.exc_info()
-                        msg = (
-                            '%s: %s; namespace=%s key=%s requires a value parseable by %s' % (
-                                exc_info[0].__name__,
-                                str(exc_info[1]),
+                        msg = [
+                            '%s: %s' % (exc_info[0].__name__, str(exc_info[1])),
+                            'namespace=%s key=%s requires a value parseable by %s' % (
                                 use_namespace,
                                 key,
                                 qualname(parser)
                             )
-                        )
+                        ]
```

Comment:
Why not use ``.format()`` here? E.g.

```python
msg = [
    '{class_name}: {error_str}'.format(
        class_name=exc_info[0].__name__,
        error_str=str(exc_info[1]),
    ),
    'namespace={namespace} key={key} requires '
    'a value parseable by {parser}'.format(
        namespace=use_namespace,
        key=key,
        parser=qualname(parser)
    )
]
```

That's much longer but it names each part of the string, which I believe is a good thing considering that ``exc_info[0].__name__`` and ``str(exc_info[1])`` are not very explicit.